Plugins/DSLCompletion/DSLComplete.cs

```diff
@@ -255,7 +255,8 @@ private string FindSelection(Check isAllowedChar, bool checkRight = true)
             
             var text = sci.GetLine(sci.CurrentLine);
 
-            if (!text.Contains("\"")) return null;
+            var style = sci.BaseStyleAt(sci.CurrentPos);
+            if (style != 6 && style != 7) return null;
```

Comment:
Note that you may prefer to use `XML.DOUBLESTRING` and `XML.SINGLESTRING` from `ScintillaNet.Lexers`. Although FD codebase is full of just using the numeric value...